diff --git a/modules/console/build.gradle.kts b/modules/console/build.gradle.kts
new file mode 100644
index 0000000..f34d260
--- /dev/null
+++ b/modules/console/build.gradle.kts
@@ -0,0 +1,11 @@
+plugins {
+    alias(libs.plugins.kotlin.jvm)
+}
+
+dependencies {
+}
+
+kotlin {
+    jvmToolchain(17)
+    explicitApi()
+}
diff --git a/modules/console/src/main/kotlin/tech/jamalam/console/Console.kt b/modules/console/src/main/kotlin/tech/jamalam/console/Console.kt
new file mode 100644
index 0000000..0548c77
--- /dev/null
+++ b/modules/console/src/main/kotlin/tech/jamalam/console/Console.kt
@@ -0,0 +1,4 @@
+package tech.jamalam.console
+
+class Console {
+}
diff --git a/modules/console/src/main/kotlin/tech/jamalam/console/LibC.kt b/modules/console/src/main/kotlin/tech/jamalam/console/LibC.kt
new file mode 100644
index 0000000..ea26336
--- /dev/null
+++ b/modules/console/src/main/kotlin/tech/jamalam/console/LibC.kt
@@ -0,0 +1,4 @@
+package tech.jamalam.console
+
+interface LibC {
+}
diff --git a/modules/console/src/main/kotlin/tech/jamalam/console/Msvcrt.kt b/modules/console/src/main/kotlin/tech/jamalam/console/Msvcrt.kt
new file mode 100644
index 0000000..53a7ee2
--- /dev/null
+++ b/modules/console/src/main/kotlin/tech/jamalam/console/Msvcrt.kt
@@ -0,0 +1,4 @@
+package tech.jamalam.console
+
+class Msvcrt {
+}
